ddtrace/profiling/exporter/http.py

```diff
@@ -85,12 +88,17 @@ def _validate_enpoint(instance, attribute, value):
         raise InvalidEndpoint("Endpoint is empty")
 
 
+def _validate_api_key(instance, attribute, value):
+    if not formats.validate_api_key(value):
+        _LOG.warning("Invalid API Key")
```

Comment:
I missed this before, if it is just a warning log message... probably fine to have this